doc/source/notebooks/advanced/varying_noise.ipynb

```diff
@@ -16,9 +16,9 @@
     "$$f(\\cdot) \\sim \\mathcal{GP}\\big(0, k(\\cdot, \\cdot)\\big)$$\n",
     "$$y_i | f, x_i \\sim \\mathcal N\\big(y_i; f(x_i), \\sigma^2_i\\big)$$\n",
     "\n",
-    "We'll demonstrate two methods. In the first demonstration, we'll assume that the noise variance is known for every data point. We'll incorporate the known noise variances $\\sigma^2_i$ into the data matrix $\\mathbf Y$, make a likelihood that can deal with this structure, and implement inference using Variational GPs with natural gradients (#TODO no natural gradients in GPflow 2 yet). \n",
+    "We'll demonstrate two methods. In the first demonstration, we'll assume that the noise variance is known for every data point. We'll incorporate the known noise variances $\\sigma^2_i$ into the data matrix $\\mathbf Y$, make a likelihood that can deal with this structure, and implement inference using variational GPs with natural gradients (#TODO no natural gradients in GPflow 2 yet). \n",
```

Comment:
Natgrads now in gpflow2? @stefanosele 

@akhayes see here it's "data points" not datapoints as well ;)

Reply:
@st-- , yes, natgrads were ported long time ago

Reply:
But this notebook still needs updating with (reverting to) the natgrads-example, then?

Reply:
I don't know much about this notebook. But looks like, yes. It requires some finishing (different PR of course)